odoo/models.py

```diff
@@ -2025,14 +2040,14 @@ def _read_group_having(self, having_domain, query):
         return expr, params, fnames_used
 
     @api.model
-    def _read_group_orderby(self, order, groupby, query):
+    def _read_group_orderby(self, order, groupby_terms, query):
```

Comment:
Maybe time to add a small docstring with input parameters, especially the type of 'groupby_terms' changes.